app/controllers/store_controller.rb

```diff
@@ -10,5 +10,9 @@ def index
       @products = Product.order(:title)
     end
   end
+
+  def categories
+    @categories = Category.includes(:sub_categories).where(parent_id: nil).all
```

Comment:
what will the difference b/w and which one is better?

`Category.includes(:parent)` vs `Category.includes(:sub_categories)`